src/Components/fe-login/FeLogin.js

```diff
@@ -37,14 +48,36 @@ export class FeLogin extends LocalizeMixin(LitElement) {
       return;
     }
 
-    const loginData = serializedValue;
-    this.dispatchEvent(
-      new CustomEvent('input-validation', { detail: loginData })
-    );
+    this.login(ev);
+    ev.target.reset();
+  }
+
+  async login(ev) {
+    this.isError = false;
+    try {
+      const { serializedValue } = ev.target;
+      await FeServices.postRequest({
+        url: '/login',
+        data: {
+          login: serializedValue.login,
+        },
+      });
+      this.dispatchEvent(
+        new CustomEvent('input-validation', { detail: serializedValue })
```

Comment:
why its need for dispatch ? and why u pass the login details, what is the use ?

Reply:
where is the response data ?, we need the data based on that only account call will trigger

Reply:
Made the changes and uploaded